internal/cli/atlas/dbusers/delete.go

```diff
@@ -52,6 +52,10 @@ func DeleteBuilder() *cobra.Command {
 		Aliases: []string{"rm"},
 		Short:   "Delete a database user for your project.",
 		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":         "username",
+			"usernameDesc": "Username that this resource deletes from the MongoDB database.",
```

Comment:
[q] is `that this resource` how we refer to the mongocli calls usually? I'd be more inclined to say:
```suggestion
			"usernameDesc": "Username to be deleted from the MongoDB database.",
```

Same with the other cases, unless resource is a standard way we phrase usually.

Reply:
I have used the same description that we have in the documentation of the endpoint for all these parameters
See https://docs.atlas.mongodb.com/reference/api/database-users-delete-a-user/
<img width="808" alt="Screenshot 2021-07-14 at 12 22 52" src="https://user-images.githubusercontent.com/5663078/125606512-94ca0158-b65d-4f75-bb7e-d6c11a411c13.png">

Reply:
sgtm, thanks

Reply:
I know that's how we say it in the API, and I know that this is a wrapper around the API, but I think Colm's right here: I wouldn't call this a "resource". It might be better to reference the old handwritten reference docs when looking for copy (if the handwritten doc exists). You can find it in [version 16](https://docs.mongodb.com/mongocli/v1.16/reference/atlas/dbuser-delete/) of the mongocli docs -- thought admittedly those aren't always great either. But that's why I'm here!

```suggestion
			"usernameDesc": "Username to delete from the MongoDB database.",
```